.eslintrc.yml

```diff
@@ -36,6 +36,7 @@ rules:
   camelcase:
     - 2
     - properties: always
+      ignoreGlobals: true
```

Comment:
This is way better than disabling in the files, well thought of. I will add it to the Lint PR if you don't mind...